wrapper/CSharp/wolfSSL-TLS-Client/wolfSSL-TLS-Client.cs

```diff
@@ -54,9 +54,20 @@ private static void clean(IntPtr ssl, IntPtr ctx)
     /// <param name="x509_ctx">Certificate in WOLFSSL_X509_STORE_CTX format</param>
     private static int myVerify(int preverify, IntPtr x509_ctx)
     {
-        /* Use the provided verification */
+        int verify = preverify;
+
+        /* example for overriding an error code */
+        /* X509_STORE_CTX_get_error API can be enabled with
+         * OPENSSL_EXTRA_X509_SMALL or WOLFSSL_EXTRA */
+        int error = wolfssl.X509_STORE_CTX_get_error(x509_ctx);
+        const int ASN_BEFORE_DATE_E = -150;  /* ASN date error, current date before */

```

Comment:
Is it possible to pull in the `error-crypt.h` values of `enum wolfCrypt_ErrorCodes` into `wolfSSL.CSharp` instead of using magic numbers here?